src/main.js

```diff
@@ -64,11 +56,16 @@ async function onGoogleLoad() {
   // of the app: https://developers.google.com/identity/sign-in/web/reference
   await new Promise((resolve) => window.gapi.load('auth2', resolve));
   try {
+
+    
   // Initializes it with the correct client ID
-    await window.gapi.auth2.init({
-      client_id: process.env.VUE_APP_CLIENT_ID.concat(
-        '.apps.googleusercontent.com',
-      ),
+    await window.gapi.auth2.init(
+      {
+      // client_id: process.env.VUE_APP_CLIENT_ID.concat(
+      //   '.apps.googleusercontent.com',
+      // ),
+      
+      client_id: settings.google_client_id
```

Comment:
How do we know that settings is initialized before this runs? What if the settings request happens to take a long time? Then this might get run before we have the settings variable. I would recommend sending the request within this try block.

Reply:
Good feedback. I thought about putting it there but I would like to have the settings be global (part of the Vue prototype) which isn't available in this block.